src/main/java/fr/lunatech/timekeeper/gauges/TimeEntriesNumberPerHoursGauge.java

```diff
@@ -64,6 +61,5 @@ public Long numberOfOtherHoursEntries() {
         return otherHours;
     }
 
-    void init(@Observes StartupEvent event) {
```

Comment:
This method souldn't be removed, I checked that with @tkuy, and he explained to me that this method is usefull for Quarkus, otherwise it didn't understand that it has to create an instance and then can't find the metric.

Reply:
We may add a comment on it to explain why we should keep it, to make sure no one will remove it later.

Reply:
ok thanks for reporting. I gues we should really have either a comment or something else here